glocaltokens/client.py

```diff
@@ -44,64 +44,74 @@ def __init__(
         """
         Initializes a Device. Can set or google_device or ip and port
         """
-        LOGGER.debug("Initializing new Device instance.")
+        log_prefix = f"[Device - {device_name}]"
+        LOGGER.debug(f"{log_prefix} Initializing new Device instance")
```

Comment:
You didn't enable `pylint` but it's advised to use format style arguments here to not render the string unless it will be printed.
```suggestion
        LOGGER.debug("%s Initializing new Device instance", log_prefix)
```

Reply:
@KapJI, weirdly i have ran pylint on this branch and it didn't show my any warnings here 🤔

Reply:
So before we have `pylint` enabled, can you update this logger invocations, so it will be easier to enable this `pylint` check in the future?